constellations/gemini-admin/src/location/locationEdit.js

```diff
@@ -116,35 +117,75 @@ export class LocationEditPage extends React.Component {
             <div id="view-location-edit">
                 {modalDiv}
                 <h2>{title}</h2>
-                <h4>Location ID</h4>
-                <input
+                <InputText
+                    description="Location ID"
```

Comment:
Hm I think this should be label="Location ID"
And the description is a short "Enter a location Id (e.g. wchs or home)"